### PR TITLE
include the git sha of the current checkout in the perf test results

### PR DESCRIPTION
**Pull request by SamBarker**
### Type of change

- Enhancement / new feature

### Description

Include the short commit ID in the results so we can trace where results come from.

### Additional Context

_Why are you making this pull request?_

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
- [ ] Make sure all Sonarcloud warnings are addressed or are justifiably ignored.
- [ ] Update documentation
- [ ] Reference relevant issue(s) and close them after merging
- [ ] For user facing changes, update CHANGELOG.md (remember to include changes affecting the API of the test artefacts too).

### CODE REVIEW

File: performance-tests/perf-tests.sh [hunk 1]
```diff
@@ -85,7 +86,7 @@ producerPerf() {
   docker run --rm --network ${PERF_NETWORK} "${KAFKA_TOOL_IMAGE}" \
       bin/kafka-producer-perf-test.sh --topic "${TOPIC}" --throughput -1 --num-records "${NUM_RECORDS}" --record-size "${RECORD_SIZE}" \
       --producer-props "${PRODUCER_PROPERTIES}" bootstrap.servers="${ENDPOINT}" | \
-      jq --raw-input --arg name "${TESTNAME}" '[.,inputs] | [.[] | match("^(?<sent>\\d+) *records sent" +
+      jq --raw-input --arg name "${TESTNAME}" --arg commit_id "${COMMIT_ID}" '[.,inputs] | [.[] | match("^(?<sent>\\d+) *records sent" +
```
[the remark this answers is not in the document]
Comment (SamBarker) [reply performance-tests/perf-tests.sh:89]: I didn't think the CI job could resolve the ID of the previous run so I populated it here so it's there in every results file.

[the remark this answers is not in the document]
Comment (k-wall) [reply performance-tests/perf-tests.sh:89]: I like the idea this tool stamping its results with the commit id.  Unambiguously knowing the provenance of data is always good.  Users may be using this tool outwith CI.

**Review by k-wall — APPROVED** [review]

File: performance-tests/perf-tests.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (franvila) [reply performance-tests/perf-tests.sh:89]: I think adding it to kibana json files could add an extra effort not needed on the CI side. The commit IDs can be taken from other files that are already stored as artifacts.

I like the idea to have it in local as well as it is right in this PR.

**Review by franvila — APPROVED** [review]

**Review by franvila — COMMENTED** [review]
I have added a new comment into the gitlab PR.

What I'm trying to explain is that the current changes add the `commit_id` to one single file containing that info and that's perfect.

But in Jenkins we are using the `producer.json` and `consumer.json` files created at the end of the script (that are not being modified in the current PR) and we have one file for each test case and include the `commit_id` in each single file does not seem appropriate to me.

**Comment by franvila** [comment]
LGTM after the changes done in strimzi-ci

**Pull request merged**